include/wlr/types/wlr_drm_lease_v1.h

```diff
@@ -0,0 +1,139 @@
+/*
+ * This an unstable interface of wlroots. No guarantees are made regarding the
+ * future consistency of this API.
+ */
+#ifndef WLR_USE_UNSTABLE
+#error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
+#endif
+
+#ifndef WLR_TYPES_WLR_DRM_LEASE_V1_H
+#define WLR_TYPES_WLR_DRM_LEASE_V1_H
+
+#include <wayland-server-core.h>
+
+struct wlr_backend;
+struct wlr_output;
+
+struct wlr_drm_lease_manager {
```

Comment:
Since this is tied to v1 of the protocol, this should be called something like `wlr_drm_lease_v1_manager`